frontend/src/app/components/modals/export-modal/wp-table-export.modal.ts

```diff
@@ -16,29 +16,21 @@ import {
 import {switchMap, takeWhile, map} from 'rxjs/operators';
 import {interval, Observable, Subscription} from 'rxjs';
 import {NotificationsService} from "core-app/modules/common/notifications/notifications.service";
+import {JobStatusModal} from "core-app/modules/job-status/job-status-modal/job-status.modal";
 
 interface ExportLink extends HalLink {
   identifier:string;
 }
 
-/*
+/**
 Modal for exporting work packages to different formats. The user may choose from a variety of formats (e.g. PDF and CSV).
-
-The backend may choose to provide the export right away (synchronously) or delayed. In the later case, the modal will poll the
-backend until the export is done.
-Because the modal has to deal with both cases, without knowing upfront whether the download will be delayed or not, it needs
-to treat every download expecting it to be delayed. Because of this, the modal cannot simply provide download hrefs which would
- allow the browser to download the export but rather has to first check whether an export is delayed or not, and if it is delayed,
- it has to wait until the export is ready. Because of the necessary check, the modal has a hidden link that it clicks on to perform
- the actual download once the export is ready (delayed or not).
-
 The modal might also be used to only display the progress of an export. This will happen if a link for exporting is provided via the locals.
  */
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/373b673f1a4d1e1f6026ed300f93434bb30e713b/frontend/src/app/components/modals/export-modal/wp-table-export.modal.ts#L28"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>